models/sessions/scratch/snowplow_web_sessions_this_run.sql

```diff
@@ -30,7 +30,7 @@ select
 
   {% if var('snowplow__session_stitching') %}
     -- updated with mapping as part of post hook on derived sessions table
-    a.domain_userid as stitched_user_id, 
+    cast(a.domain_userid as {{dbt_utils.type_string() }}) as stitched_user_id,
```

Comment:
I noticed during testing this doesn't actually increase the char limit for Redshift, at least when moving from the old version to the new (I haven't tested a full refresh). I am wondering is it worth writing our own version of this macro with the ability to specify the char limit

Reply:
Just tested a full refresh. Same result. Still set to 128

Reply:
I'll have a look at implementing this today in snowplow utils:
https://github.com/snowplow/dbt-snowplow-utils/issues/65